src/main/java/com/ecom/shopping/adminkart/AdminKartApplication.java

```diff
@@ -1,12 +1,12 @@
 package com.ecom.shopping.adminkart;
 
-import com.ecom.shopping.adminkart.controller.ProductCategoryController;
+import com.ecom.shopping.adminkart.dto.ProductCategoryDto;
 import org.apache.ibatis.type.MappedTypes;
 import org.mybatis.spring.annotation.MapperScan;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
-@MappedTypes({ProductCategoryController.class})
+@MappedTypes({ProductCategoryDto.class})
```

Comment:
try to remove class and then check it.

Reply:
Sir we tried but it is not working 
We put package name into the @MappedType annotation apart of Dto.class so now what we do